http-tck/src/main/java/io/micronaut/http/tck/ServerUnderTestProvider.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micronaut.http.server.tck;
+package io.micronaut.http.tck;
```

Comment:
I am concerned with this package change. It will break downstream implementations of the TCK